recipes-core/custom-nftables-rules/files/nftables-production.conf

```diff
@@ -0,0 +1,40 @@
+#!/usr/sbin/nft -f
+
+flush ruleset
+
+define BUILDER_RPC_SRC = 0.0.0.0  #TODO: Replace with actual IP address
+define LOGS_COLLECTOR_SRC = 1.1.1.1 #TODO: Replace with actual IP address
+define LOGS_COLLECTOR_PORT = 1234 #TODO: Replace with actual port
+
+table inet filter {
+    chain input {
+        type filter hook input priority 0; policy drop
+
+        # Allow established/related connections
+        ct state established,related accept
+
+        # Production mode rules
+        tcp dport 8551 accept comment "Consensus Client Engine API"
+        tcp dport 30303 accept comment "Execution Client P2P"
+        tcp dport { 8545, 8546 } accept comment "Execution Client RPC"
+        tcp dport 27017 accept comment "Searcher Input Channel"
```

Comment:
What's listening on this one?

Reply:
Based on Angela's [doc](https://www.notion.so/flashbots/BoB-external-tasks-1126b4a0d876804f924dea7b8ce1cfbc?pvs=4#11e6b4a0d876804195becec99d967cfd), it is the searcher's input channel